astropy/uncertainty/core.py

```diff
@@ -220,6 +220,19 @@ def pdf_percentiles(self, percentile, **kwargs):
             return self.distribution._new_view(percs)
         else:
             return percs
+            
+    def pdf_asymstd(self):
+        """ a convenience function that computes the 50% percentile
```

Comment:
Documentation needs to be in https://numpydoc.readthedocs.io/en/latest/format.html . Please also refer to https://astropy.readthedocs.io/en/latest/development/docguide.html .

Reply:
sorry ,i forgot about that .

Reply:
@pllim shouldnt i also add these docstring to rst files or will it automatically add during the building of the html page

Reply:
Docstring will be rendered in the API section. As for user facing narrative, I'll leave that to the main reviewer.